src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public String doWork() {
+        return "Bulldozer - started its work!";
+    }
+
```

Comment:
Wrong implementation. 

In those classes override doWork(), so it will `print message` that certain machine started its work.

Override stopWork() as well. It should `print messages` that certain machine stopped its work.